life_dashboard/tests/test_dashboard.py

```
@@ -0,0 +1,113 @@
+        self.user.refresh_from_db()
+        assert self.user.first_name == 'Selenium'
+        assert self.user.last_name == 'User'
+        assert self.user.email == 'selenium@example.com'
+
+    def test_level_up_flow(self):
+        # Set up initial state
+        self.profile.experience = 0
+        self.profile.save()
+        
+        # Create and complete a quest
+        quest = Quest.objects.create(
```

Ruff (0.11.9)

Undefined name `Quest`

(F821)